src/middlewares/requestMiddlewares.js

```diff
@@ -0,0 +1,35 @@
+import response from '../utils/response';
+import messages from '../utils/messages';
+import DbServices from '../services/dbServices';
+import models from '../models';
+
+const { User, Request } = models;
+const { getTwoTables } = DbServices;
+
+const verifyManager = async (req, res, next) => {
```

Comment:
let change verifyManager to  verifyRequestLineManager

Reply:
Done